sources/translations/zh/client.json

```diff
@@ -0,0 +1,125 @@
+{
+    "tray": {
+        "toggle": "切换",
+        "quit": "退出",
+        "mention": {
+            "one":"提及",
+            "other":"提及",
+            "zero": "",
+            "two": "",
+            "few":"",
+            "many":""
```

Comment:
Could you confirm that Chinese plural rules are the same like in English? Do you have any other rules in some cases, like for little number of things vs large amount (this happens in some languages for instance in Polish there's `few` and `many` rules, `few` are for numbers/digits like `2`,`3`,`4` and `many` describes form for basically other set of numbers/digits)? If you're unsure on that, you might as well verify it in the practice; WebCord will automatically use the rules from this config for the number that is placed before the translation.